sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-BCAG.e2e-spec.ts

```diff
@@ -0,0 +1,75 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+import { YesNoOptions } from "@sims/test-utils";
+import { CredentialType, InstitutionTypes } from "../../models";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-BCAG.`, () => {
+  it(
+    "Should determine BCAG as eligible when total assessed need is greater than 0 " +
```

Comment:
I believe that the "total assessed" must be equal or greater than 1.

Reply:
yes, the logic is `>= 1`, I thought the same is expressed when it says `greater than 0`. But I will update for the sake of consistency with the logic

Reply:
Unless we ensure that it is an integer, 0.1 would be greater than zero and "not >= 1".

Reply:
Got it 👍